daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/transaction/PartialTransaction.scala

```diff
@@ -69,27 +70,32 @@ private[lf] object PartialTransaction {
     */
   final case class Context(
       info: ContextInfo,
+      minChildVersion: TxVersion,
```

Comment:
Some documentation here might be good. I think there are two cases in particular that are important:
1. Childrens of rollback nodes, see my comment below.
2. No children at all -> 1.dev. We could also use an `Option` to make that explicit but I don’t feel strongly here either way so your choice.

Reply:
added some comments to help explain what is going on